### PR TITLE
:lipstick: #73 - style: use @maykin-ui/admin-ui for list view styling

### PR DESCRIPTION
**Pull request by svenvandescheur**
Ready for merge

### CODE REVIEW

File: ui/src/views/DashboardView/DashboardView.tsx [hunk 1]
```diff
@@ -37,7 +43,7 @@ const DashboardView = () => {
   const { loading: loadingZaaktypen, error: errorZaaktypen } = useAsync(async () => {
     if (!selectedCatalogus) return;
 
-    const zaaktypenEndpoint = `catalogi/zaaktypen/?status=alles${
+    const zaaktypenEndpoint = `catalogi/zaaktypen/?status=alles&page=1${
```
Comment (SilviaAmAm) [inline ui/src/views/DashboardView/DashboardView.tsx:46]: Is this in preparation for this endpoint to support pagination? Should it maybe be added later?

File: ui/src/views/DashboardView/DashboardView.tsx [hunk 2]
```diff
@@ -54,86 +60,87 @@ const DashboardView = () => {
...
-        useFlexGap
-        position={'sticky'}
-        top={64}
-        zIndex={1}
-        sx={{
-          pb: 2,
-          backgroundColor: 'white',
-          borderBottom: '1px solid',
-          borderColor: 'divider',
+    <>
```
Comment (SilviaAmAm) [inline ui/src/views/DashboardView/DashboardView.tsx:88]: Is this one needed? I think everything is inside the `List` tag?

**Review by SilviaAmAm — APPROVED** [review]

File: ui/src/views/DashboardView/DashboardView.tsx (hunk 1 above)
Comment (svenvandescheur) [reply ui/src/views/DashboardView/DashboardView.tsx:46]: I think this is a mistake, thanks for noticing!

File: ui/src/views/DashboardView/DashboardView.tsx (hunk 2 above)
Comment (svenvandescheur) [reply ui/src/views/DashboardView/DashboardView.tsx:88]: I think you're right

**Pull request merged**